### ISSUE TITLE
Yaml file got overridden

### ISSUE THREAD

**Issue by Lumpusz** [esphome/issues#1095]
**Operating environment/Installation (Hass.io/Docker/pip/etc.):**

Hass.io from Docker
**ESP (ESP32/ESP8266, Board/Sonoff):**

ESP 32 wrover kit
**Affected component:**

Yaml editor

**Description of problem:**
I was editing and uploading configuration repeatedly as I was doing some refactoring, and one time the config got overridden with the load text, and I lost it entirely. I don't remember explicitly hitting save, I just hit "Edit" from the upload dialog then I quickly hit "Upload" from the edit dialog, as the errors were in an included '.h' file. Now it just has this:
```
Loading configuration yaml...
```
Whenever the yaml file is being loaded, no actions should be allowed. Especially not this, as one can loose the entire config for good.

**Problem-relevant YAML-configuration entries:**
```yaml
PASTE YAML FILE HERE

```

**Logs (if applicable):**

```
PASTE DEBUG LOG HERE

```

**Additional information and things you've tried:**

**Comment by jonathanadams** [comment]
It's likely this has been fixed in esphome/esphome#1025.

**Comment by glmnet** [comment]
Yes, as a lot of changes have been made to the dashboard by @jonathanadams we better close this, thanks for your report @Lumpusz if you see some strange behavior please report again.
